### PR TITLE
Update 08-plot-ggplot2.Rmd

### PR DESCRIPTION
**Pull request by sushmitavgopalan16**
added more details to the explanations for answers to challenges 4a, 4b and 5

this is in response to issue # 586

### CODE REVIEW

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 1]
```diff
@@ -259,6 +259,9 @@ variables and their visual representation.
 > > ## Solution to challenge 4a
 > >
 > > Here a possible solution: 
+> > Notice that the `color` argument is supplied outside of the aes() function.
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd]: Add code formatting (backtics) to `aes()`

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 2]
```diff
@@ -278,7 +281,11 @@ variables and their visual representation.
 > > ## Solution to challenge 4b
 > >
 > > Here a possible solution: 
-> >
+> > Notice that supplying the color argument inside the aes() functions enables you to
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd]: Add code formatting to `aes()` and `color` (and remove single quotes from the latter).

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 3]
```diff
@@ -389,7 +396,9 @@ code to modify!
 > > ## Solution to Challenge 5
 > >
 > > Here a possible solution: 
-> >
+> > xlab() and ylab() set labels for the x and y axes, respectively
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd]: Add code formatting to `xlab()`, `ylab()`, and `theme()`.

**Comment by jcoliver** [comment]
Great additions @sushmitavgopalan16 . Would you update the PR with revisions proposed above (add back-tic code formatting)?

**Comment by sushmitavgopalan16** [comment]
Thanks @jcoliver ! I've made the changes you requested!

**Comment by jcoliver** [comment]
Thanks, @sushmitavgopalan16 !

**Pull request merged**